### ISSUE TITLE
Upload Feedback

### ISSUE THREAD

**Issue by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#158]
For larger uploads that take longer than the current checklist animation, add in a money counter. 

Has to be in the right currency.

**Comment by g105b** [comment]
Uploading a statement is the first point of interaction for new users. Can we always make a 100% correct guess on the currency for each of the supported types of upload?

**Comment by richardbirkin** [comment]
@g105b 

yes i think we can. currency is given in every upload type we've seen so far. 

Maybe we put this as a loveable addition to Milestone 8 (currencies) ?

**Comment by g105b** [comment]
Yes!
